pkg/template/controller/templateinstance_controller.go

```diff
@@ -51,9 +53,11 @@ const readinessTimeout = time.Hour
 // using its own service account, first verifying that the requester also has
 // permissions to instantiate.
 type TemplateInstanceController struct {
-	restmapper     meta.RESTMapper
-	config         *rest.Config
-	templateClient templateclient.Interface
+	restmapper        meta.RESTMapper
```

Comment:
why do you have two restmappers.

Reply:
same as below, we're using a different one for readiness checking.  possibly it should all be switched to the dynamic mapper + dynamic(json?) client.

Reply:
> same as below, we're using a different one for readiness checking. possibly it should all be switched to the dynamic mapper + dynamic(json?) client.

You can keep the extra config, but onlyi one restmapper. Compiled restmappers are being removed in 1.11.

Reply:
restmapper removed.  dynamic mapping for everyone.